### ISSUE TITLE
Mirror Symmetry is displacing some (all?) parts when scaled.

### ISSUE THREAD

**Issue by Lisias** [net-lisias-ksp/TweakScale#167]
TL;DR:

1. Load  the Osprey Stock craft on the SPH.
2. Make sure ScaleChain is active (make things easier to detect)
3. Scale up one point the Cockpit.
4. Check the nose, the rear landing gear and the lower tanks.

![screenshot48](https://user-images.githubusercontent.com/64334/108003964-26858800-6fd3-11eb-80d3-5fbeac8c4488.png)

On a educated guess, something is borking the attachment nodes scaling....

![screenshot49](https://user-images.githubusercontent.com/64334/108004273-2c2f9d80-6fd4-11eb-9d88-793727242c37.png)

![screenshot50](https://user-images.githubusercontent.com/64334/108004102-9267f080-6fd3-11eb-88c4-aa228206da72.png)

Reproduced on KSP 1.11.1 (probably works the same on every version, but it's nice to check). No Exceptions or suspicious log entries were found on the [KSP.log.zip](https://github.com/net-lisias-ksp/TweakScale/files/5984826/KSP.log.zip) (attached for completude).

**Comment by Lisias** [comment]
Just for info, TweakScale 2.5.0.x (Beta) also have the problem. This may be an unimplemented Use Case I allowed to pass through....

The misbehaviour was reproduced too on 1.4.5, eliminating the influence of the environment. It's TweakScale code for sure.

**Comment by Lisias** [comment]
Moving to 2.4.4.7, as I need to rush 2.4.4.6 into production due KSP 1.11.2

**Comment by Lisias** [comment]
**THAT WAS UNEXPECTED!!** :)

Commit https://github.com/net-lisias-ksp/TweakScale/commit/df0fef22ba67816f910663bae2009803664adbd7 fixed all the issues, but this one:

![screenshot90](https://user-images.githubusercontent.com/64334/142715677-dc3937fe-3baa-4b1c-885f-6fd96d0fdff6.png)

Out of despair, as I just wasn't understand where in fscking hell I could be doing that mistake, I removed the fuel tanks and rebuilt them - just because, I just was wiling to do something with some resemblance of usefulness.

And from that point, things worked fine!!!!

![screenshot92](https://user-images.githubusercontent.com/64334/142715834-d54ab4e5-bd30-49f5-88a5-a616d268f69c.png)

Reloading the original craft many times and retrying things, I reached to this misbehaviour:

![screenshot93](https://user-images.githubusercontent.com/64334/142715879-7ded018a-4e54-4888-a47f-50f8ec2a43a4.png)

Some of the mk0 fuek tanks **weren't set on mirroring**, besides the central mk0, the fins and the gears being set to mirror allright.

Interesting enough, these mk0 tanks that "lingered" on the other side are exactly the ones borking here. Using that "tainted" parts to rebuild the tanks reproduced the problem later, I had to take **new** parts from the Parts Palette in order to make a behaving craft! :)

So I concluded that part of this issue wasn't on TweakScale's shoulder, but something on KSP itself (perhaps a bug on the KSP build used to build the craft? It could be created on a interim, buggy, KSP build perhaps).

Anyway…. Case closed! :)
